### ISSUE TITLE
Profil windows opens when using network following tool 

### ISSUE THREAD

**Issue by varrieta** [QGEP/QGEP#555]
When using the follow network tools, the windows for the profil opens. Is it really necessary?

**Comment by urskaufmann** [comment]
in the moment only for the button "select current path". This button should be placed somewhere else.

**Comment by olivierdalang** [comment]
> in the moment only for the button "select current path". This button should be placed somewhere else.

@varrieta @urskaufmann Right, so it's currently not completely useless to open the profile pane. If we were to move the "select current path" button, where would it be best to put it ? I'm not sure it's a good idea to overcrowd the toolbar with too many buttons...

I suggest to add it as a "dropdown" under the upstream/downstream buttons, would that make sense ?

**Comment by urskaufmann** [comment]
In the moment, this button only works after a network-following. But there is another possibility in the button to configure the select. This configure is to select catchment areas over the different fk-fields. This button will then be probably renamed to "extend selection to connected elements" and it should be possible to start with any existing selection of reaches.
Suggestion: make a dropdown-bar with upstream, downstream, lenght-profile, then we have new place for (dropdown-)button "extend selection" or " select current path".

**Comment by varrieta** [comment]
A dropdown button seems good to me so the profil windows open only if we need it.
